cookbooks/aws-parallelcluster-platform/test/controls/disable_services_spec.rb

```diff
@@ -25,7 +25,7 @@
   end
 end
 
-control 'tag:config_services_disabled_on_amazon_family' do
+control 'tag:testami_tag:config_services_disabled_on_amazon_family' do
```

Comment:
should be: `tag:testami` not `tag:testami_tag` right?

Reply:
It's not `tag:testami_tag`, it's `tag_testami`_`tag_config`, because it will run both on Test phase of the AMI build and as part of the checks we run on the configured node (daily Kitchen tests).